modules/shacl/src/main/scala/es/weso/shacl/SHACLPrefixes.scala

```diff
@@ -32,6 +32,7 @@ object SHACLPrefixes {
   lazy val sh_oneOrMorePath: IRI = sh + "oneOrMorePath"
 
   lazy val sh_class : IRI= sh + "class"
+  lazy val sh_rootClass : IRI= sh + "rootClass"
```

Comment:
I agree as I said [here](https://github.com/labra/shaclex/issues/41). I see that the TopQuandrant SHACL implementation have [rootClass](http://shacl.org/playground/dash.ttl) constraint under dash (http://datashapes.org/dash#) namespace.
I have no idea about how this is handled usually. I would suggest to use shext (http://www.w3.org/ns/shacl/ext#).

Thoughts ?